src/integrations/YattIntegrationHelpers.js

```diff
@@ -1,16 +1,27 @@
 import { IPC_HANDLERS, IPC_FUNCTIONS } from "../modules/constants";
-
 import axios from "axios";
 import dayjs from "dayjs";
 
 export default {
-  getHeaders(credential) {
-    let authHeader;
-    authHeader = `Bearer ${credential.accessToken}`;
-    return {
-      Authorization: authHeader,
-      Accept: "application/json",
-    };
+  async getHeaders(credential) {
+    let authHeader = { headers: {} };
+    if (credential.type === "bearer")
+      authHeader = {
+        headers: {
+          Authorization: `Bearer ${credential.accessToken}`,
+          Accept: "application/json",
+        },
+      };
+    else if (credential.type === "cookie")
+      authHeader = {
+        headers: {
+          Authorization: `Bearer ${credential.accessToken}`,
```

Comment:
@dockabernathy01 We shouldn't have a bearer token in the headers for cookie auth.